corehq/apps/userreports/pillow.py

```diff
@@ -180,16 +189,21 @@ def rebuild_table(self, sql_adapter):
         self._processor.rebuild_table(sql_adapter)
 
 
-def get_kafka_ucr_pillow(pillow_id='kafka-ucr-main'):
+def get_kafka_ucr_pillow(ucr_division='0f', pillow_id='kafka-ucr-main'):
     return ConfigurableReportKafkaPillow(
         processor=ConfigurableReportPillowProcessor(
             data_source_provider=DynamicDataSourceProvider(),
             auto_repopulate_tables=False,
+            ucr_division=ucr_division
         ),
         pillow_name=pillow_id,
     )
 
 
+get_kafka_ucr_pillow_08 = partial(get_kafka_ucr_pillow, '08')
+get_kafka_ucr_pillow_9f = partial(get_kafka_ucr_pillow, '9f')
```

Comment:
Instead of defining these statically we could add a `params` field to the pillowtops settings and pass that through to the constructor. Then we can just decide in localsettings who to partition for each env.